CHANGELOG.md

```diff
@@ -12,4 +17,4 @@ for Alpine, CentOS and Ubuntu
 For detailed information on the tasks carried out during this release, please see the GitHub milestone
 [v7.0.0.1](https://github.com/wso2/docker-ei/milestone/7).
 
-[v7.0.0.1]: https://github.com/wso2/docker-ei/compare/v6.5.0.3...v7.0.0.1
```

Comment:
It should be [v7.0.0.2]: https://github.com/wso2/docker-ei/compare/v7.0.0.1..v7.0.0.2 ?.